### DISCUSSION TITLE
ESBMC web interface

### DISCUSSION THREAD

**Discussion by monicamarcus** [esbmc/esbmc#675] [Q&A] [answered]
Thanks for making ESBMC public! 

Before I would download it on my MacBook Pro computer, I thought I should try ESBMC from the web interface. There are 2 examples already available to try, but none works, either on Chrome or on Safari browsers. The complain is about syntax, it does not recognize '<' sign.

Unknown Error:SyntaxError: Unexpected token < in JSON at position 0

What should I do?

Thank you,
Monica Marcus

**Comment by kaled-alshmrany** [comment, marked as answer]
First, many thanks for your interest in using our web interface. Regarding the problem you faced, I guess there is something happening in your browser because I have tried it on my 2 devices, and it worked well on both (see attachments). I suggest reopening the browser and making sure to set the following options:
1) --unwind nr  (50 <)
2) --16 --32 --64   set width of machine word (default is 64) 

Best regards

![device_2](https://user-images.githubusercontent.com/48326737/163239817-3361f7d2-7736-40d1-aeac-d62b83387b8c.jpg)
![Screen Shot 2022-04-13 at 8 39 46 PM](https://user-images.githubusercontent.com/48326737/163239841-88fed601-8c40-4dbe-8548-68e7c2b17fb2.png)
![Screen Shot 2022-04-13 at 8 40 28 PM](https://user-images.githubusercontent.com/48326737/163239845-bb70f35c-4202-42cb-93d0-aafe2da639bd.png)

**Reply by monicamarcus** [reply]
Thank you for the quick reply.

Now, after setting explicitly the unwind nr., example 2 works, but only on my old Mac computer. I tried again on a new macOS computer (using the M1 Apple chip) and I get the same syntax error.

On the old mac the verification for the example 2 fails, however. I think it is due to this assertion failing:

(_Bool)(j == i)

I see it uses Boolector. Should I use another solver?

Best regards,
Monica

**Reply by monicamarcus** [reply]
Indeed, the assertion (_Bool)(j == i) after the loop should fail. I got it!
